src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CatalogueSolutionsController.cs

```diff
@@ -67,5 +70,77 @@ public async Task<ActionResult> UpdateAsync(string orderId)
             await _orderRepository.UpdateOrderAsync(order);
             return NoContent();
         }
+
+        [HttpGet]
+        [Route("{orderItemId}")]
+        public ActionResult<OrderItemModel> GetOrderItem(string orderId, string orderItemId)
+        {
+            var orderItemKey = $"{orderId}_{orderItemId}";
+
+            if (CatalogueSolutionOrderItems.ContainsKey(orderItemKey))
+            {
+                return CatalogueSolutionOrderItems[orderItemKey];
+            }
+            else
+            {
+                return new OrderItemModel
+                {
+                    serviceRecipientModel = new ServiceRecipientModel
+                    {
+                        OdsCode = "OX3"
+                    },
+                    CatalogueItemId = orderItemId,
```

Comment:
According to the Json contract this should be SolutionId I think, @sam-roberts-bjss is that right?

Reply:
We had a call where i thought it was agreed it would be CatalogueItemId,

Reply:
Ok no worries, I'll update the Json contracts

Reply:
I've just spoke to @alibombaye & he is concern the context might get a little confusing if start changing the name of that ID. The context of the endpoint suggests it would like a solution ID so I would like to suggest we leave it as 'solutionId'.

Reply:
ok